### ISSUE TITLE
Upgrading to from jsxstyle 1.02 to 2.0.6 cause react-modal errors

### ISSUE THREAD

**Issue by StokeMasterJack** [smyte/jsxstyle#96]
After updating jsxstyle from 1.02 to 2.0.6 I now gets a bunch react-modal errors in the js console:

Warning: react-modal: App element is not defined. Please use `Modal.setAppElement(el)` or set `appElement={el}`. This is needed so screen reades don't see main content when modal is opened. It is not recommended, but you can opt-out by setting `ariaHideApp={false}`.

**Comment by meyer** [comment]
I suspect this error is related to #94. Can you swap import syntax and let me know how it goes?

There have been a few changes between jsxstyle 1 and jsxstyle 2. If you haven’t already, give the README a once-over: https://github.com/smyte/jsxstyle/tree/103c076/packages/jsxstyle#readme

**Comment by StokeMasterJack** [comment]
Actually I did a rollback of the jsx upgrade and I'm still getting the react-modal errors. It's strange because we were not getting those before. The only thing that changed was that I updated my version of yarn and re-ran yarn install. No way to roll that back since node_modules doesn't get added to git. So apparently rebuilding the node_modules folder from package.json is not deterministic.

Anyway, not a jsxstyle issue. Thanks. Sorry for wasting your time.
